tests/collections/asr/numba/rnnt_loss/test_rnnt_pytorch.py

```diff
@@ -0,0 +1,247 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import pytest
+import torch
+
+from nemo.collections.asr.parts.numba.rnnt_loss.rnnt_numpy import RNNTLoss as RNNTLoss_Numpy
+from nemo.collections.asr.parts.numba.rnnt_loss.rnnt_pytorch import RNNTLossNumba
+
+DEVICES = ['cpu']
+
+if torch.cuda.is_available():
```

Comment:
Can we test if the output of two losses give the same output?

Reply:
Yup I test that in [test_case_small_random](https://github.com/NVIDIA/NeMo/pull/1854/files#diff-4453c997c448a541e49a86954b761850507284b04f8478e86879ff9765fdf7afR95-R107) and [test_case_large_random](https://github.com/NVIDIA/NeMo/pull/1854/files#diff-4453c997c448a541e49a86954b761850507284b04f8478e86879ff9765fdf7afR226-R243)